packages/cli-kit/src/public/node/system.ts

```diff
@@ -190,3 +191,10 @@ export function terminalSupportsPrompting(): boolean {
 export function isCI(): boolean {
   return isTruthy(process.env.CI)
 }
+
+/**
+ * Check if the current environment is a WSL environment.
+ *
+ * @returns True if the current environment is a WSL environment.
+ */
+export const isWsl = wsl
```

Comment:
The `is-wsl` package does some work on import, including opening files. I think we should wrap it up in a function (so, use `await import...`) to avoid extra start-up cost